torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15700,6 +15700,8 @@ def error_inputs_mean(op_info, device, **kwargs):
                         DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit',),
                         # TODO: FIXME tolerance is too high
                         DecorateInfo(unittest.skip('Skipped!'), 'TestGradients'),
+                        # Pre-existing condition - (nan vs nan) comparison
+                        DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_backward'),
```

Comment:
Is the problem just that nan, when compared with nan, returns False?

Reply:
Yes. 

I think `assert_close` has an option to specify `nan` against `nan` as equal. Should we set it to `True`? I think it should be ok as for this test we care about both (subclass path and regular tensor path) values being equal. (Correctness of backward is verified in other test).

Reply:
Yup. On it. Thanks!